crates/ruma-common/src/events/room/message/reply.rs

```diff
@@ -1,253 +1,77 @@
 use std::fmt;
 
-use indoc::formatdoc;
-
 use super::{FormattedBody, MessageType, OriginalRoomMessageEvent};
 
-pub fn get_plain_quote_fallback(original_message: &OriginalRoomMessageEvent) -> String {
-    let sender = &original_message.sender;
+const HTML_ESCAPE_CHARACTERS: &[(char, &str)] = &[
+    ('&', "&amp;"),
+    ('<', "&lt;"),
+    ('>', "&gt;"),
+    ('"', "&quot;"),
+    ('\'', "&apos;"),
+    ('\n', "<br />"),
```

Comment:
In HTML5, the preferred way to write this is just `<br>`. The spec isn't specific about an HTML version for `org.matrix.html`, but I think it's okay to assume that (besides, anything parsing regular HTML as opposed to XHTML can handle `<br>` just fine).

```suggestion
    ('\n', "<br>"),
```